src/components/NavBar/index.tsx

```diff
@@ -1,34 +1,37 @@
 import Link from "next/link";
 import React from "react";
-import Styles from "./NavBar.module.css" 
+import Styles from "./NavBar.module.css";
 const NavBar = () => {
-
-    return (
-        <>
-        <div className={Styles.body}>
-            <span className={Styles.logo}>
-            <Link href="/Home">JIMO</Link>
-            </span>
-            <div className={Styles.navLinksContent}>
-            <span className={Styles.homeNav}>
-            <Link href="/Home" >Home</Link>
-            </span>
-            <span className={Styles.aboutusInfo}>
-            <Link href="#" >AboutUs</Link>
-            </span>
-            <span className={Styles.PriceInfo}>
-            <Link href="#" >Pricing</Link>
-            </span>
-            <span className={Styles.featuresInfo}>
-            <Link href="#" >Features</Link>
-            </span>
-            <span className={Styles.contactInfo}>
-            <Link href="#" >Contact</Link>
-            </span>
-            </div>
+  return (
+    <>
+      <div className={Styles.body}>
```

Comment:
Make use of semantic HTML tags, the right way to create a navbar is by using the `<nav>` tag
here is a semantic example of a navbar (modify accordinly):

```html
<nav>
  <ul>
    <li><a href="/about">About</a></li>
    <li><a href="/contact">Contact</a></li>
    <li><a href="/press">Press</a></li>
  </ul>
</nav>
```